paymentsheet/src/test/java/com/stripe/android/customersheet/data/DefaultCustomerSessionElementsSessionManagerTest.kt

```diff
@@ -288,6 +324,7 @@ class DefaultCustomerSessionElementsSessionManagerTest {
         )
 
         assertThat(errorReporter.getLoggedErrors()).containsExactly(
+            ErrorReporter.SuccessEvent.CUSTOMER_SHEET_CUSTOMER_SESSION_ELEMENTS_SESSION_LOAD_SUCCESS.eventName,
             ErrorReporter
                 .UnexpectedErrorEvent
                 .CUSTOMER_SESSION_ON_CUSTOMER_SHEET_ELEMENTS_SESSION_NO_CUSTOMER_FIELD
```

Comment:
We expect that both of these are fired. We should indicate if `elements/session` was loaded successfully from the endpoint but the unexpected error indicates to us that the response is malformed.